.circleci/config.yml

```
@@ -219,10 +219,16 @@ jobs:
                 --output-file "$CIRCLE_TEST_REPORTS/eslint/results.xml" \
                 $FILES_TO_LINT
             fi
+  translate:
+    <<: *defaults
+    parallelism: 1
+    steps:
+      - prepare
       - run:
           name: Build calypso-strings.pot
           when: always
           command: |
+            sudo apt install gettext
```

I'd expect this to be something like the following:

```suggestion
            sudo apt-get update -yqq
            sudo apt-get install -yqq gettext
```

This could easily take loads of time. Caching might help, but the best solution is probably to build a custom Docker image with this dependency included.

Not a blocker or something that needs to go into this PR, just making a note.